app/pathogen/arbovirus/dashboard/(banner)/ArboBanner.tsx

```diff
@@ -0,0 +1,37 @@
+"use client";
+
+import React, { useContext } from "react";
+import { ArboContext } from "@/contexts/arbo-context/arbo-context";
+import { Button } from "@/components/ui/button";
+import { download, generateCsv, mkConfig } from "export-to-csv";
+
+interface ArboBannerProps {
+    className?: string;
+  }
+
+export const ArboBanner = ({className}: ArboBannerProps) => {
+    const state = useContext(ArboContext);
+    
+    const downloadData = () => {
+        const csvConfig = mkConfig({
+            useKeysAsHeaders: true,
+            filename: "arbotracker_filtered_dataset",
+          });
+          let csv = generateCsv(csvConfig)(state.filteredData);
+          download(csvConfig)(csv);
+    }
+
+    return (
+      <div className={className}>
```

Comment:
I followed the example of the other components and put a lot of the styling choices for this component on the outside in page.tsx, but maybe this is confusing? For example the side by side text button layout here comes from the outside of this component? Maybe a bad idea.

Reply:
Yeah I agree this is not something we want to do --> would've made sense if it was a customizable or reusable component both of which it is not. Can create a ticket to quickly address that post launch -- ok for now and good callout!